utils/firestore.js

```diff
@@ -826,37 +825,34 @@ const filterDB = async (queries, siteCode, isParent) => {
 
     // Direct the generation and execution of queries based on the search properties present. If neither firstName nor lastName are present, this function is bypassed.
     const handleNameQueries = async (firstNameQuery, lastNameQuery, phoneEmailQuery) => {
-        if (firstNameQuery) {
-            // const fNameStringQueryForSearch = generateQuery(firstNameQuery, 'string');
-            // await executeQuery(fNameStringQueryForSearch);
+        const promises = [];
```

Comment:
Variable `promises` could be renamed to `searchPromises`.

Reply:
Good call. Updated.